### ISSUE TITLE
A not-free book listed as free

### ISSUE THREAD

**Issue by johnfrombluff** [jphall663/awesome-machine-learning-interpretability#10]
Firstly, thanks! This page is _awesome_.

I really don't like to look a gift horse in the mouth, but  the book "An Introduction to Machine Learning Interpretability" by Gill and Hall is not free. It's paywalled: you need a Safari account to access it, which isn't free.

Best wishes,

John

**Comment by jphall663** [comment]
Argh ... they keep changing it. So for now I changed the link to a free link - you do have to provide an email ... and I moved it to a different subject heading.

I'm going to keep this issue open b/c the Dataiku sponsorship will end eventually ... maybe sooner rather than later ...

What I like about this booklet is that it's a non-technical resource, so I would like to wait to take it down until it can be replaced with another free, broad, non-technical resource - if you are aware of one, that would help. Personally, I can live with people signing up for a free trial of Safari to get the booklet. (As a co-author I don't profit in anyway, that I am aware of, in that scenario.)
